L1Trigger/L1THGCal/python/customHistoSeeding.py

```diff
@@ -13,7 +13,7 @@ def set_histomax_seeding_params(parameters_seeding_c3d,
                         nBins_X1,
                         nBins_X2,
                         binSumsHisto,
-                        seed_threshold,
+                        seed_threshold
```

Comment:
Here and below in this file: if possible it would be better to keep the formatting of the existing code unchanged.